src/nvim/ex_cmds.h

```diff
@@ -26,6 +28,37 @@ typedef struct {
   list_T *additional_elements;  ///< Additional data left from ShaDa file.
 } SubReplacementString;
 
+
+// Defs for inc_sub functionality
+
+// List of matches in a line
+#define _noop(x)
+KLIST_INIT(colnr_T, colnr_T, _noop)
```

Comment:
I checked that out a bit, it would mean hand-writing the for-loops, which is certainly doable. Also, I'd have to deallocate manually in the case of `MatchedLine`, since there's no way to initialize with a custom deallocator as with klist.

Should I still do it? I think I can manage, but I'm uncertain if deallocating manually is a good tradeoff.